src/ruby/SphericalCoordinates.i

```diff
@@ -75,9 +75,6 @@ namespace gz
                 /// This has kept a bug for backwards compatibility, use
                 /// LOCAL2 for the correct behaviour.
                 LOCAL = 4,
-
-                /// \brief Heading-adjusted tangent plane (X, Y, Z)
-                LOCAL2 = 5
```

Comment:
there's a `GZ_UTILS_WARN_IGNORE__DEPRECATED_DECLARATION` macro earlier in this file. Can that be removed?